test/specs/modules/Accordion/Accordion-test.js

```diff
@@ -77,6 +77,29 @@ describe('Accordion', () => {
       wrapper
         .should.have.state('activeIndex', -1)
     })
+    it('is called with (event, index) on AccordionTitle click', () => {
```

Comment:
Thanks for this test. Could we also add one more test similar to this that asserts that the correct title/content are "active" given an `activeIndex` of 0, 1, and 2?  Something like this added to `describe('activeIndex')`, note I did not test this so it could have issues:

```jsx
it('sets the correct pair of title/content active', () => {
  const wrapper = shallow(
    <Accordion>
      <Accordion.Title />
      <Accordion.Content />
      <Accordion.Title />
      <Accordion.Content />
      <Accordion.Title />
      <Accordion.Content />
    </Accordion>
  )
  wrapper.setProps({ activeIndex: 0 })
  wrapper.childAt(0).should.have.prop('active', true)
  wrapper.childAt(1).should.have.prop('active', true)

  wrapper.setProps({ activeIndex: 1 })
  wrapper.childAt(2).should.have.prop('active', true)
  wrapper.childAt(3).should.have.prop('active', true)

  wrapper.setProps({ activeIndex: 2 })
  wrapper.childAt(4).should.have.prop('active', true)
  wrapper.childAt(5).should.have.prop('active', true)
})
```